src/modules/organisation/persistence/organisation.repository.integration-spec.ts

```diff
@@ -754,10 +755,11 @@ describe('OrganisationRepository', () => {
                 DoFactory.createOrganisationAggregate(false, { name: 'AnotherTest', kennung: 'KENNUNG2' }),
                 DoFactory.createOrganisationAggregate(false, { name: 'TestName2', kennung: 'DIFFERENTKENNUNG' }),
             ];
-
+            /* eslint-disable no-await-in-loop */
             for (const organisation of organisations) {
```

Comment:
Das kannst du direkt umbauen Promise.all(...)

Reply:
as discussed before we don't need to address seeding